demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -152,25 +151,34 @@ def prepare_hooks(
             "all_files": self.all_files,
             "input_mode": self.input_mode,
         }
-        PyclnHook(**hooks["pycln"], **kwargs).prepare_hook(PYTHONPATH)
-        RuffHook(**hooks["ruff"], **kwargs).prepare_hook(
+        PyclnHook(**hooks.pop("pycln", None), **kwargs).prepare_hook(PYTHONPATH)
+        RuffHook(**hooks.pop("ruff", None), **kwargs).prepare_hook(
             self.python_version_to_files, IS_GITHUB_ACTIONS
         )
```

Comment:
will passing None to these hooks work as expected? Shouldnt we check if we got these hooks in the yml?

Reply:
meaning 
```
hook = hooks.pop(ruff,None)
if hook: 
    ruffHook(.....
```